tests/conftest.py

```
@@ -171,7 +146,7 @@ def docker_services() -> Generator[None, None, None]:
 
 
 @pytest.fixture
-def redis_client(docker_services: None) -> Generator[Any]:
+def redis_client(docker_services: None) -> Generator[Any, None, None]:
```

Ruff (0.14.0)

Unused function argument: `docker_services`

(ARG001)